apps/allocations/app/components/App/AllocationsHistory.js

```diff
@@ -117,21 +120,31 @@ const Amount = styled.div`
   font-weight: 600;
 `
 
-// const Recipient = styled.div`
-//   display: flex;
-//   flex-direction: column;
-// `
-//
-// const RecipientProgress = styled.div`
-//   margin-top: 7px;
-//   margin-bottom: 4px;
-// `
-//
-// const RecipientAmount = styled.div`
-//   color: ${({ theme }) => theme.contentSecondary};
-//   align-self: flex-end;
-//   font-size: 12px;
-// `
+const Recipient = styled.div`
+   display: flex;
+   flex-direction: row;
+   width: 100%;
+   padding: 10px 10px 4px;
+   border-radius: 4px;
+   background-color: white;
+   box-shadow: 0 2px 4px #dde4e9;
```

Comment:
Is there an aragon color that uses that `#dde4e9` code? We follow a convention of using named colors from aragon-ui themes always when they exist, so it allows updating any palette change seamlessly.

Reply:
In the future I will do this. However, in this case I'll just remove the css entirely since it doesn't fit with @jayalaves design.